src/utils.js

```diff
@@ -184,3 +184,15 @@ export function clearClientArrays(client) {
 
   return client;
 }
+
+export function convertClientIdToName(clientId, knownClients) {
+  const clients = knownClients || [];
```

Comment:
Just curious on why not use default params?

Reply:
Great point. Improved in e5d9619.